app/controllers/concerns/sorting.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Sorting
+  def sort_collection(items, default_sort, default_direction = :asc)
```

Comment:
Keyword arguments for 'default' parameters:

```suggestion
  def sort_collection(items, default_sort:, default_direction: :asc)
```